src/backend/tests/unmocked/reimbursement-requests.test.ts

```diff
@@ -0,0 +1,49 @@
+import { alfred } from '../test-data/users.test-data';
+import ReimbursementRequestService from '../../src/services/reimbursement-requests.services';
+import { AccessDeniedException } from '../../src/utils/errors.utils';
+import { createFinanceTeamAndLead, createTestReimbursementRequest, resetUsers } from '../test-utils';
+import prisma from '../../src/prisma/prisma';
+
+describe('Reimbursement Requests', () => {
+  beforeEach(async () => {
+    await createFinanceTeamAndLead();
+  });
+
+  afterEach(async () => {
+    await resetUsers();
+  });
+
+  test('Delete Reimbursement Request fails when deleter is not a finance lead', async () => {
+    const reimbursement = await createTestReimbursementRequest();
+    if (!reimbursement) {
+      console.log('Failed to create Reimbursement');
+      assert(false);
+    }
+    await expect(() =>
+      ReimbursementRequestService.deleteReimbursementRequest(reimbursement.reimbursementRequestId, alfred)
+    ).rejects.toThrow(
+      new AccessDeniedException(
+        'You do not have access to delete this reimbursement request, only the creator or finance leads can delete a reimbursement request'
```

Comment:
update the throw to say whatever you changed it to above ('...only finance leads ...'), and add the test cases I mentioned earlier